### DISCUSSION TITLE
Path in documentation needs updating

### DISCUSSION THREAD

**Discussion by mpm-meto** [dtcenter/METplus#2441] [Existing Builds] [answered]
The path to the obs error table in 
https://met.readthedocs.io/en/latest/Users_Guide/ensemble-stat.html
 **MET_BASE/data/table_files/obs_error_table.txt**
does not seem correct. I could only find it under MET_BASE/share/met/table_files?

**Comment by CPKalb** [comment, marked as answer]
The table in MET_BASE/share/met/table_files is the correct file.  It is wrong in the documentation and needs to be updated.  We have updated the develop branch of the [documentation](https://met.readthedocs.io/en/develop/Users_Guide/ensemble-stat.html#ensemble-stat-configuration-file) so that this now points to the correct location and it will be fixed for the next release.

Christina
